src/popup/App.vue

```diff
@@ -127,18 +127,30 @@
         </div>
         <hr />
 
-        <div v-if="existing.length > 0" class="text-center">
+        <div v-if="existing.length > 0">
           <p class="font-weight-bold">Or use an existing alias</p>
-          <div v-for="alias in existing" v-bind:key="alias">
-            <span class="text-info">{{ alias }}</span>
-            <button
-              v-if="alias"
-              v-clipboard="() => alias"
-              v-clipboard:success="clipboardSuccessHandler"
-              v-clipboard:error="clipboardErrorHandler"
-              class="btn btn-success btn-sm copy-btn"
-            >Copy</button>
-          </div>
+          <table class="table table-sm table-borderless">
+            <colgroup>
+              <col span="1" style="width: 80%;" />
+              <col span="1" style="width: 20%;" />
+            </colgroup>
+            <tbody>
+              <tr v-for="alias in existing" v-bind:key="alias">
+                <td>
+                  <a v-b-tooltip.hover v-bind:title="alias" class="small-text">{{ alias | truncate(50, "...") }}</a>
```

Comment:
The UI is much nicer, 👍. Do you know if we can enable the "copy on click", meaning when user clicks on the alias, the alias will be copied to the clipboard. This way user doesn't have to click on the "copy" button which is quite small.